src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public Ball getRandomBall() {
+        int number = new Random().nextInt(100);
```

Comment:
make this instance variable instead of recreating Random object each time method's called